code/modules/events/ion_storm.dm

```diff
@@ -19,18 +20,26 @@
 	if(announceEvent == ION_ANNOUNCE || (announceEvent == ION_RANDOM && prob(ionAnnounceChance)))
 		GLOB.event_announcement.Announce("Ion storm detected near the station. Please check all AI-controlled equipment for errors.", "Anomaly Alert", 'sound/AI/ionstorm.ogg')
 
-
 /datum/event/ion_storm/start()
 	//AI laws
 	for(var/mob/living/silicon/ai/M in GLOB.living_mob_list)
-		if(M.stat != 2 && M.see_in_dark != 0)
+		if(M.stat != DEAD && M.see_in_dark != FALSE)
 			var/message = generate_ion_law(ionMessage)
 			if(message)
 				M.add_ion_law(message)
 				to_chat(M, "<br>")
 				to_chat(M, "<span class='danger'>[message] ...LAWS UPDATED</span>")
 				to_chat(M, "<br>")
 
+	for(var/mob/living/silicon/robot/R in GLOB.living_mob_list)
+		if(R.stat != DEAD && !R.connected_ai && !R.emagged)
+			var/message = generate_ion_law(ionMessage)
```

Comment:
Objection.
This seems like it would give ERT borgs permanent, unremovable ion laws.
It also seems like it is buffing the ion storm event too much. Not just making it silent, but also making it affect unlinked borgs.

Reply:
True I didnt think of that. I'll just remove this part of the PR as in most situations a cyborg will have a linked AI and this could cause too many issues/edge cases with stuff like ERT borgs... maybe even syndicate ones now that I think about it.

Updating now.